scripts/determine-reboot-cause

```diff
@@ -218,6 +220,10 @@ def main():
         sonic_logger.log_error("User {} does not have permission to execute".format(pwd.getpwuid(os.getuid()).pw_name))
         sys.exit("This utility must be run as root")
 
+    if os.path.exists(REBOOT_PROCESSED_FILE):
```

Comment:
@anamehra can you add `RemainAfterExit=true` in the `determine-reboot-cause.service`, that should ensure systemd starts this service only once (unless someone manually starts the service, not the point of this PR)